cypress/fixtures/osd-gcp/OsdCuratedGCPClusterProperties.json

```diff
@@ -0,0 +1,170 @@
+{
+  "Clusters": [
+    {
+      "ClusterName": "ocmui-cypress-smoke-osd-curated-marketplace-sa-gcp",
+      "Type": "OSD",
+      "CloudProvider": "Google Cloud Platform (GCP)",
```

Comment:
Not sure this matters here, but we are changing the GCP name in the other ticket so maybe just call this Google Cloud. (And other places in this file)